.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!-- What does this implement/fix? Explain your changes here. -->
+
+
+PR Checklist
+------------
+ - [ ] Tests?
+ - [ ] Docs?
+ - [ ] CHANGELOG updated?
+ - [ ] Issue raised/referenced?
```

Comment:
Maybe we split it like this? Because we do not want it in every PR an update of the change log

```suggestion

Contributor (creator of PR) checklist
-------------------------------------
 - [ ] Tests updated (for new features and bugfixes)?
 - [ ] Documentation updated (for new features)?
 - [ ] Issue referenced (for PRs that solve an issue)?
 
For Reviewer
------------
  - [ ] CHANGELOG updated if important change?

```